src/helpers/inputValidationHelpers.js

```diff
@@ -0,0 +1,30 @@
+import * as constants from '../constants/InputValidation.constants.js';
+
+export const validateName = (value) => {
+    console.log(value);
```

Comment:
Also don't leave console.log when merging to master

Reply:
Will address